shared/cfg/machines.cfg

```diff
@@ -16,6 +16,14 @@ variants:
             cd_format = ahci
         enable_pvpanic = yes
         ioport_pvpanic = 0x505
+        pci_controllers = root_port pcie_switch dmi2pci_bridge pci_bridge
+        pci_bus = pci.0
```

Comment:
Will make all virtio pci device connect to downstream port, USB and AHCI contorller connected to pcie.0 and other PCI device connect to pci-bridge.